shm/cargo.hpp

```diff
@@ -16,6 +16,12 @@ class Cargo {
         return *this;
     }
 
+    bool operator==(const Cargo& freight2) {
+        if (name_ == freight2.name_)
```

Comment:
```suggestion
        return name_ == freight.name_;
```

Reply:
Great! Could you please change freight to freight2.name_ ? Actually freight2 is variable name, then I'll accept that ! :)